bower.json

```diff
@@ -1,23 +1,23 @@
 {
   "name": "purescript-optics",
-  "version": "0.3.0",
```

Comment:
In the future, don't worry about updating the `version` field. The gulp tasks take care of that.